deltachat-ios/Controller/ChatViewController.swift

```diff
@@ -786,6 +790,14 @@ extension ChatViewController: MessagesDataSource {
         }
     }
 
+    private func sendVoiceMessage(url: NSURL) {
+        DispatchQueue.global().async {
+            let msg = DcMsg(viewType: DC_MSG_AUDIO)
```

Comment:
that should be DC_MSG_VOICE, the difference is mainly that for voice messages the filename is not shown and they are not mixed with DC_MSG_AUDIO (which are eg. mp3 songs) if dc_get_next_media() is used at some point.

Reply:
Ups, iOS ignores that differnence at the moment (wrt. the filename)